lib/airbrake/shoryuken/error_handler.rb

```diff
@@ -0,0 +1,34 @@
+module Airbrake
+  module Shoryuken
+    ##
+    # Provides integration with Shoryuken.
+    class ErrorHandler
+      def call(_worker, _queue, _sqs_msg, body)
+        yield
+      rescue => e
+        notify_airbrake(e, notice_context(body))
+        raise e
+      end
+
+      private
+
+      def notice_context(body)
+        body.is_a?(Array) ? { batch: body } : { body: body }
+      end
+
+      def notify_airbrake(exception, context)
+        return unless (notice = Airbrake.build_notice(exception, context))
+
```

Comment:
@kyrylo I dunno what to add as action. I can try the class name as the given example.